### PR TITLE
Pristine source typo fix, add a few missing git ignores

### PR DESCRIPTION
**Pull request by nyetwurk**
Reproduce with:

mkdir build; cd build; ../configure; make

### CODE REVIEW

**Comment by nyetwurk** [comment]
Currently investigating if I can convince libtool to not cock this up whilst crosscompiling.

File: xml/Makefile.am [hunk 1]
```diff
@@ -31,7 +31,7 @@ RNGdir			= $(dtddir)
 
 # Sorted list of available numeric RNG versions,
 # extracted from filenames like NAME-MAJOR[.MINOR][.MINOR-MINOR].rng
-RNG_numeric_versions    = $(shell ls -1 *.rng \
+RNG_numeric_versions    = $(shell echo $(wildcard $(top_srcdir)/xml/*.rng) \
```
Comment (kgaillot) [inline xml/Makefile.am]: The Travis errors show problems with the XML schemas, so I suspect this wildcard isn't a drop-in replacement. You probably need to strip off the leading path in the sed.

Comment (nyetwurk) [inline xml/Makefile.am]: xml/versions.rng is identical ... it looks like something else is going wrong

Comment (nyetwurk) [inline xml/Makefile.am]: The regression test scripts do not work correctly...
```
Relax-NG parser error : xmlRelaxNGParse: could not load /home/travis/build/ClusterLabs/pacemaker/xml/pacemaker-1.0.rng
```

It should be looking in mkbuild/xml, not the pristine source directory.

Comment (nyetwurk) [inline xml/Makefile.am]: Looks like the problem is "ls -1" includes linefeeds where wildcard does not. I am in the process of rebasing and will submit a new PR

**Pull request closed without merging**